packs/osx-attacks.conf

```diff
@@ -537,6 +537,19 @@
       "version" : "2.8.0",
       "description" : "MaMi OSX Malware 2017-01-12 (https://objective-see.com/blog/blog_0x26.html)",
       "value" : "bogus certificate added to key store by this malware"
+    },
+    "Behavioral_Reverse_Shell": {
+      "query" : "SELECT DISTINCT(processes.pid), processes.parent, processes.name, processes.path, \
```

Comment:
Why the distinct? Is there a concern with multiple process entires for a single pid?